lib/cdo/hamburger.rb

```diff
@@ -113,12 +113,27 @@ def self.get_hamburger_contents(options)
       entry[:title] = I18n.t("#{loc_prefix}#{entry[:title]}")
     end
 
-    if  options[:language] == "en"
+    if options[:language] == "en"
       if options[:user_type] == "teacher"
         teacher_entries << {
           title: I18n.t("#{loc_prefix}professional_learning"),
           url: CDO.studio_url("/my-professional-learning"),
         }
+
+        teacher_entries << {
+          title: I18n.t("#{loc_prefix}incubator"),
+          url: CDO.studio_url("/incubator"),
+        }
+      elsif options[:user_type] == "student"
```

Comment:
nit: Do these need to be conditionally added? Could we just iterate over all entries and add the incubator entry, similar to how international is added below?

Reply:
They are all the same entry, right? I agree with doing something similar to how international is added.

Reply:
Good call!